coalib/bearlib/aspects/Smell.py

```diff
@@ -136,15 +136,16 @@ class docs:
                               Integer age, String gender,
                               String occupation, String city){
 
-            System.out.printf("Welcome %s %s, a %d-year-old %s "\
-"from %s who works as a%s\n",firstName, lastName, age, gender, city, \
-occupation);
+            System.out.printf(
+              "Welcome %s %s, a %d-year-old %s from %s who works as a%s",
+              firstName, lastName, age, gender, city, occupation
+            );
         }
         """
         example_language = 'java'
         importance_reason = """
-        Data clumps make codes difficult to read, debug, scale, and also
-        hardly reusable.
+        Data clumps make code difficult to read, understand, and reuse.
+        It also spoils the organization.
```

Comment:
"the organization"? Like e.g. coala association e.V.? I guess you mean the organization of the code base, I'd rather speak of architecture here. I had to think a long time until I got what you meant here ^^